### ISSUE TITLE
New user login page

### ISSUE THREAD

**Issue by mishrraG** [nic-delhi/AarogyaSetu_Android#185]
When the user installs the application for the first time and tries to log in giving his her phone number , the field already contains the country code (+91) and the suggested numbers already have a (+91 prefix). <br> Enhancement required 
[label: ]

**Comment by vasthava** [comment]
Since this application is mainly intended to use by indians i believe prefixing with +91 is  convenient and user friendly

**Comment by mishrraG** [comment]
Yes I agree but the main issue I'm stating is that recommended numbers which pop up (via Google Recommendations) contain the prefix +91 already appended before the number and users tend to just click on the desired number.

Consider this:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;      **|+91--------|**
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;      **|+91--------|** <- number to selected
+91 ------------

After clicking  

+91 **+91[first 7 characters of phone number]**

This often leads to an error and is not a good practice to have a constant placeholder(+91).
And even if we are keeping the placeholder constant , we might as well keep the last 10 characters of the string input (the phone number) irrespective of the fact that the user specifies the number with/without the country code in the start.
